contact.html

```diff
@@ -14,41 +15,47 @@
 <!-- nav -->
 
   <ul class="nav nav-pills nav">
-    <li role="presentation" class="active"><a href="#">Home</a></li>
+    <li role="presentation" class="active"><a href="index.html">Home</a></li>
     <li role="presentation"><a href="#">Profile</a></li>
```

Comment:
I'd remove the nav links that don't go anywhere and make sure the ones you keep have their proper file paths